cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -182,16 +182,17 @@ class ScannerCommand : CliktCommand(name = "scan", help = "Run external license
         outputDir.safeMkdirs()
         writeOrtResult(ortResult, outputFiles, "scan")
 
-        val scanResults = ortResult.scanner?.results
+        val scannerRun = ortResult.scanner
```

Comment:
Commit message:
- Maybe be a bit more specific in the title and say "model: Move issue-related code from `ScanRecord` to `ScannerRun`"
- "prepeares" -> "prepares"